app/components/addTme/addTmeController.js

```diff
@@ -0,0 +1,21 @@
+'use strict'
+angular.module('batApp')
+    .controller('addTmeController', function (getDataFactory, $state) {
+      var vm = this
+      vm.err = false
+
+      vm.save = function (data) {
+        getDataFactory.addTme().save(data).$promise
+                .then((response) => {
+                  if (!response.error) {
+                    $state.go('menuTemplate.tmeViewTable')
+                  } else {
+                    vm.err = true
+                    vm.errMsg = response.error.message
```

Comment:
we need to have global error messages. not in every api.   

in interceptor need to capture error message and show in all api error messsages